modules/jaeger-install.adoc

```diff
@@ -0,0 +1,43 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-installation.adoc
+////
+
+[id="jaeger-operator-install_{context}"]
+= Installing the Jaeger Operator
+
+To install Jaeger you use the link:https://operatorhub.io/[OperatorHub] to install the Jaeger Operator.
```

Comment:
Is this really the right way to install it? It looks to me like the only thing at that link is community jaeger. Can we get rid of this sentence and write a new intro? I also think that the sentence isn't helpful. It says to install jaeger, go to the link to install jaeger.

If this is a valid install path, can the sentence be changed to something like this?
s/install Jaeger you use the link:https://operatorhub.io/[OperatorHub] to install the Jaeger Operator.
/install Jaeger visit the link:https://operatorhub.io/[OperatorHub]. Click the Jaeger Operator and click Install./

If not can we get rid of it?